.github/workflows/ci.yml

```diff
@@ -40,16 +40,31 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      - name: Build awx_devel image for running checks
+        uses: ./.github/actions/awx_ci_setup
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Run check ${{ matrix.tests.name }}
-        run: AWX_DOCKER_CMD='${{ matrix.tests.command }}' make github_ci_runner
+        run: AWX_DOCKER_CMD='${{ matrix.tests.command }}' make docker-runner
 
   dev-env:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
 
+      - name: Build awx_devel image for running checks
+        uses: ./.github/actions/awx_ci_setup
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Would it make sense to have `run_awx_devel` include `awx_ci_setup?`

While we do want to run `awx_ci_setup` independently sometimes, I don't think we'd ever want to run `run_awx_devel` without depending on `awx_ci_setup`.